programs/stake/src/stake_instruction.rs

```diff
@@ -47,7 +47,7 @@ fn get_optional_pubkey<'a>(
     )
 }
 
-pub const DEFAULT_COMPUTE_UNITS: u64 = 750;
+pub use solana_sdk::stake::instruction::DEFAULT_COMPUTE_UNITS;
```

Comment:
Is there a way we could mark these public-ness of these uses as deprecated? Seems like sdk is a more natural place to expose the public interface, instead of the program crate

Reply:
My intuition is that compute cost are something for user programs to have access to, so yes could go in the SDK.

@t-nelson Thoughts whether the compute costs of built-in / core programs should be in the SDK?

Reply:
right, I'd think users would like to know the cost of native programs/instructions when they construct txs, in theory at least. I vaguely recall there were efforts to keep everything in program's own crate, but hope that doesn't apply to static things like default cost or program id.

Reply:
remove program crates' public-ness at fde290f88f094eaabdbb4c492f83b2db3ac870aa